Command/EmailSubmissionsCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace EMS\CoreBundle\Command;
+
+use EMS\CommonBundle\Command\CommandInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use EMS\CoreBundle\Service\Form\Submission\FormSubmissionService;
+use Symfony\Component\Console\Command\Command;
+use EMS\CoreBundle\Service\MailerService;
+
+class EmailSubmissionsCommand extends Command implements CommandInterface
+{
+    /** @var FormSubmissionService */
+    protected $formSubmissionService;
+
+    /** @var LoggerInterface */
+    protected $logger;
+
+    /** @var MailerService */
+    protected $mailerService;
+
+    protected static $defaultName = 'ems:submissions:email';
+
+    const TITLE = 'Form submissions';
+
+    public function __construct(FormSubmissionService $formSubmissionService, LoggerInterface $logger, MailerService $mailerService)
+    {
+        $this->formSubmissionService = $formSubmissionService;
+        $this->logger = $logger;
+        $this->mailerService = $mailerService;
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this->setDescription('Send a list of form submissions to the specified email address or addresses')
+            ->addArgument(
+                'emails',
+                InputArgument::IS_ARRAY | InputArgument::REQUIRED,
+                'To which email addresses do you want to send the list?'
+            )
+            ->addOption(
+                'formInstance',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'From which form instance do you want to mail submissions? Defaults to all.'
+            );
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $emails = (array) $input->getArgument('emails');
+        $formInstance = \strval($input->getOption('formInstance'));
```

Comment:
```suggestion
        $formInstance = (string) $input->getOption('formInstance');
```

Reply:
PHPStan does not allow this - Cannot cast array<string>|bool|string|null to string.